nucliadb_sdk/src/nucliadb_sdk/v2/sdk.py

```diff
@@ -521,7 +525,7 @@ def prepare_request(
             elif not isinstance(content, request_type):
                 raise TypeError(f"Expected {request_type}, got {type(content)}")
             elif isinstance(content, BaseModel):
-                data = content.model_dump_json(by_alias=True, exclude_unset=True)
+                data = content.model_dump_json(by_alias=True, exclude_unset=False)
```

Comment:
Why are you changing `exclude_unset` here? This affects the ability to change public models and makes bw/c harder in nucliadb

Reply:
Because otherwise RagStrategy objects does not work properly unless I set the name of the Pydantic object. But I can force it if its a problem.